### PR TITLE
Citadel map

### PR DESCRIPTION
**Pull request by Vladislav94**

### CODE REVIEW

**Comment by andkoval** [comment]
@Vladislav94 Please fix all warnings.
Screenshot 2022-01-15 at 14 54 20

File: src/data/guilds.js [hunk 1]
```diff
@@ -368,7 +373,6 @@ const guilds = [
         members: [
             '0x9f89Fe18A7AD4458eEa63Aad762aA8586E82eeD6',
             '0xbD6F5BdC401AB1Ca811e40755f4A2ddAd75cE2cc',
-            '0x17a3831B39cc557296d0C322f9C2D42C0b3A8F3f',
```
[the remark this answers is not in the document]
Comment (andkoval) [reply src/data/guilds.js:371]: duplicated address

**Review by andkoval — CHANGES_REQUESTED** [review]

File: src/api/thegraph.js [hunk 2]
```diff
@@ -282,6 +282,25 @@ export default {
...
+            allRealm = [...allRealm, ...realm];
+        }
+        return allRealm;
+    },
+
+    async getRealmById(id) {
+        return await this.getRealmData(parselQuery(id)).then((response) => {
+            return response.data.parcel
+        })
+        // return await getGraphData(clientFactory.realmClient, parselQuery(id)).then(response => response);
```
[the remark this answers is not in the document]
Comment (Vladislav94) [reply src/api/thegraph.js]: done

File: src/components/Citadel/Citadel.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            setSelectedParcel(parcel);
+        });
+    }, [selectedId]);
+
+    useEffect( () => {
+        // let controller = new AbortController();
+        setTimeout( () => {
+            if( initialize ) initCitadel();
+        }, 100);
+        // return () => controller?.abort(); // cleanup on destroy
```
[the remark this answers is not in the document]
Comment (Vladislav94) [reply src/components/Citadel/Citadel.js]: done

File: src/contexts/ClientContext.js [hunk 4]
```diff
@@ -182,9 +183,10 @@ const ClientContextProvider = (props) => {
         setLoadingRealm(true);
 
         thegraph.getRealmByAddress(address).then((response) => {
-            let [filter, dir] = getFilter(realmFilter);
+            // let [filter, dir] = getFilter(realmFilter);
 
-            setRealm(commonUtils.basicSort(response, filter, dir));
+            // setRealm(commonUtils.basicSort(response, filter, dir));
```
[the remark this answers is not in the document]
Comment (Vladislav94) [reply src/contexts/ClientContext.js]: done

**Review by andkoval — APPROVED** [review]

**Pull request merged**